classifiers/reference_quality/tiktoken_length_classifier/config.py

```diff
@@ -0,0 +1,46 @@
+from util.configs import build_classifier_function_config
+from util.enums import State, BricksVariableType, RefineryDataType, SelectionType
+from . import tiktoken_length_classifier, INPUT_EXAMPLE
+
+
+def get_config():
+    return build_classifier_function_config(
+        # strapi information
+        function=tiktoken_length_classifier,
+        input_example=INPUT_EXAMPLE,
+        issue_id=347,
+        tabler_icon="ListNumbers",
+        min_refinery_version="1.7.0",
```

Comment:
Yes that is correct. If the next version is going to be 2.0.0 I will update this. Thanks! 👍

Reply:
Quick note, since this will be set to a version refinery doesn't have yet this testing in refinery needs to be tricked. Either by locally (during testing) change the version to 1.xx or change the html element in the ui (via the inspector)